tests/test_elastic.py

```diff
@@ -763,3 +770,80 @@ def test_multiple_migrations_with_delete(self):
             var = self.es.indices.get_aliases()
             assert_equal(var[settings.ELASTIC_INDEX + '_v{}'.format(n + 1)]['aliases'].keys()[0], settings.ELASTIC_INDEX)
             assert not var.get(settings.ELASTIC_INDEX + '_v{}'.format(n))
+
+class TestSearchFiles(SearchTestCase):
+
+    def setUp(self):
+        super(TestSearchFiles, self).setUp()
+        self.node = ProjectFactory(is_public=True, title='Otis')
+        self.osf_storage = self.node.get_addon('osfstorage')
+        self.root = self.osf_storage.get_root()
+
+    def test_search_file(self):
+        self.root.append_file('Shake.wav')
+        find = query_file('Shake.wav')['results']
+        assert_equal(len(find), 1)
+
+    def test_delete_file(self):
+        file = self.root.append_file('I\'ve Got Dreams To Remember.wav')
```

Comment:
As mentioned above, `file` is a reserved word in python; change the variable name here, and in the many other usages in these tests. (I won't add a comment for each but will check later)